spot-ml/src/main/scala/org/apache/spot/proxy/ProxySuspiciousConnectsModel.scala

```diff
@@ -188,4 +235,30 @@ object ProxySuspiciousConnectsModel {
     ipWordDF.rdd.map({ case Row(ip, word) => ((ip.asInstanceOf[String], word.asInstanceOf[String]), 1) })
       .reduceByKey(_ + _).map({ case ((ip, word), count) => SpotLDACInput(ip, word, count) })
   }
+
+  def ipWordCountFromDFSpark(sc: SparkContext,
```

Comment:
what's with the distinction? everything in the ML routines is wrtitten in spark
maybe a name change and some scaladoc explaining the difference....